docs/building-apps.md

```diff
@@ -8,22 +8,23 @@ order: 4
     enact pack [options]
 
   Options
-    -p, --production  Build in production mode
-    -i, --isomorphic  Use isomorphic code layout
-                      (includes prerendering)
-    -w, --watch       Rebuild on file changes
-    -l, --locales     Locales for isomorphic mode; one of:
+    -p, --production     Build in production mode
+    -i, --isomorphic     Use isomorphic code layout
+                         (includes prerendering)
+    -w, --watch          Rebuild on file changes
+    -l, --locales        Locales for isomorphic mode; one of:
             <comma-separated-values> Locale list
             <JSON-filepath> - Read locales from JSON file
             "none" - Disable locale-specific handling
             "used" - Detect locales used within ./resources/
             "tv" - Locales supported on webOS TV
             "signage" - Locales supported on webOS signage
             "all" - All locales that iLib supports
-    -s, --snapshot    Generate V8 snapshot blob
-                      (requires V8_MKSNAPSHOT set)
-    -c, --custom-skin Build with a custom skin
-    --stats           Output bundle analysis file
+    -s, --snapshot       Generate V8 snapshot blob
+                         (requires V8_MKSNAPSHOT set)
+    -c, --custom-skin    Build with a custom skin
+    -ch, --content-hash  Add the hash of the chunk
```

Comment:
Please update the description with the latest one.
Let's move it to the top.

Reply:
OK